### PR TITLE
Added Russian as an option for Authenticator

### PR DESCRIPTION
**Pull request by retrQJS**
#### Description of changes

Added a Russian translation for the authenticator.

#### Description of how you validated changes
I validated these changes through my two years of Russian that I took, which provided me a simple basis for the language.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ErikCH** [comment]
Thanks @retrQJS for adding this! We will take a look :)

**Review by wlee221 — CHANGES_REQUESTED** [review]
Hi @retrQJS, really appreciate your contribution! Few things to complete translation:

1. Can you re-export `ruDict` in `packages/ui/src/i18n/dictionaries/authenticator/index.ts`:

https://github.com/aws-amplify/amplify-ui/blob/193cdc91915fb33bb7b4671b203e84ba45091a93/packages/ui/src/i18n/dictionaries/authenticator/index.ts#L15-L16

2. Can you re-export `ruDict` in  `packages/ui/src/i18n/dictionaries/index.ts`: 

https://github.com/aws-amplify/amplify-ui/blob/193cdc91915fb33bb7b4671b203e84ba45091a93/packages/ui/src/i18n/dictionaries/index.ts#L16-L18

3. re-export `ruDict` in `packages/ui/src/i18n/translations.ts`: 

https://github.com/aws-amplify/amplify-ui/blob/193cdc91915fb33bb7b4671b203e84ba45091a93/packages/ui/src/i18n/translations.ts#L29-L31

https://github.com/aws-amplify/amplify-ui/blob/193cdc91915fb33bb7b4671b203e84ba45091a93/packages/ui/src/i18n/translations.ts#L91-L92

4. Update docs to mention that Russian is now supported ✨ 



https://github.com/aws-amplify/amplify-ui/blob/193cdc91915fb33bb7b4671b203e84ba45091a93/docs/src/pages/components/authenticator/customization.i18n.web.mdx

*note*: supported languages are listed alphabetically under English.

**Comment by wlee221** [comment]
Please feel free to use https://github.com/aws-amplify/amplify-ui/pull/1496 as refrence!

**Comment by retrQJS** [comment]
> Please feel free to use #1496 as refrence!

Hello, thank you for the reference. I was able to make these changes in my forked repository, but I am unsure of how to add those commits to this pull request. I am fairly new to GitHub and was just wondering if there was a way to combine pull requests? 

Thank you for your patience

**Review by wlee221 — DISMISSED** [review]
Looking good, thanks! We'll just have one more person check the translations and merge 🙏

**Review by ErikCH — DISMISSED** [review]

File: packages/ui/src/i18n/dictionaries/authenticator/rs.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+export const ruDict = {
```
Comment (wlee221) [inline packages/ui/src/i18n/dictionaries/authenticator/rs.ts:1]: Ooh one more thing, this file name should be `ru.ts` for imports to work correctly.

Comment (retrQJS) [reply packages/ui/src/i18n/dictionaries/authenticator/rs.ts]: Done

**Review by wlee221 — APPROVED** [review]

**Review by thaddmt — APPROVED** [review]
LGTM

**Review by ErikCH — APPROVED** [review]

**Comment by wlee221** [comment]
Merged, thanks @retrQJS for quick responses!

**Pull request merged**